federation/src/watcher.c

```diff
@@ -0,0 +1,179 @@
+#include "watcher.h"
+
+spiffebundle_Watcher *spiffebundle_Watcher_New()
+{
+    spiffebundle_Watcher *watcher
+        = (spiffebundle_Watcher *) calloc(1, sizeof(*watcher));
```

Comment:
```suggestion
        = calloc(1, sizeof(*watcher));
```
Cast is not necessary